Modifiers/WeaponModifiers/VelocityDamage.cs

```diff
@@ -9,18 +9,18 @@ public class VelocityDamage : WeaponModifier
 	{
 		public override ModifierTooltipLine[] Description => new[]
 			{
-				new ModifierTooltipLine { Text = $"Added damage based on velocity (multiplier: {Math.Round(Power/2, 1)}x)", Color = Color.Lime}
+				new ModifierTooltipLine { Text = $"Added damage based on velocity (multiplier: {RoundedPower}x)", Color = Color.Lime}
```

Comment:
Might consider telling explicitly it concerns the player's velocity in this context